rules/Arguments/Rector/ClassMethod/ArgumentAdderRector.php

```diff
@@ -202,7 +204,22 @@ private function shouldSkipParameter(
                 return false;
             }
 
-            return $this->isName($node->params[$position], $argumentName);
+            $param = $node->params[$position];
+            // argument added and name has been changed
+            if (! $this->isName($param, $argumentName)) {
+                return true;
+            }
+
+            // argument added and default has been changed
+            if ($this->changedArgumentsDetector->isDefaultValueChanged(
+                $param,
+                $argumentAdder->getArgumentDefaultValue()
```

Comment:
please assign `$argumentAdder->getArgumentDefaultValue()` to variable as re-used in next check.